clinica/pipelines/deeplearning_prepare_data/info.json

```
@@ -0,0 +1,8 @@
+{
+    "id": "aramislab/deeplearning-prepare-data",
+    "author": "Mauricio Diaz",
+    "version": "0.1.0",
+    "space_caps": "42M",
+    "space_wd": "42M",
```

I noticed that the size depends of the type of extraction 🤔 I propose to select the "worst" case scenario i.e. when having heavy outputs:

```suggestion
    "space_caps": "80M",
    "space_wd": "110M",
```

Based on:
```Text
$ du -sh caps/subjects/sub-ADNI022S0004/ses-M00/deeplearning_prepare_data/image_based
 33M	caps/subjects/sub-ADNI022S0004/ses-M00/deeplearning_prepare_data/image_based
$ du -sh caps/subjects/sub-ADNI022S0004/ses-M00/deeplearning_prepare_data/patch_based
 17M	caps/subjects/sub-ADNI022S0004/ses-M00/deeplearning_prepare_data/patch_based
$ du -sh caps/subjects/sub-ADNI022S0004/ses-M00/deeplearning_prepare_data/slice_based
 78M	caps/subjects/sub-ADNI022S0004/ses-M00/deeplearning_prepare_data/slice_based
$ du -sh wd_s
111M	wd_s
```

---

Ok, I'm agree!